app/messages/Messages.scala

```diff
@@ -156,6 +161,12 @@ object Messages {
             english = s"Your Self Assessment tax bill is ${amount.gdsFormatInPounds}",
             welsh   = s"Mae’ch bil treth Hunanasesiad yn dod i gyfanswm o ${amount.gdsFormatInPounds}"
           )
+
+        case TaxRegime.Sia =>
+          Message(
+            english = s"Your Simple Assessment tax bill is ${amount.gdsFormatInPounds}",
```

Comment:
We should get this content checked - not necessarily int this ticket,  but we should note it down in that ticket we were gonna list things we were unsure of in. Same goes for the rest of the content you've added which isn't specifically mentioned on the ticket.